app/code/Magento/MediaGalleryUi/etc/acl.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Acl/etc/acl.xsd">
+    <acl>
+        <resources>
+            <resource id="Magento_Backend::admin">
+                <resource id="Magento_Backend::content">
+                    <resource id="Magento_Backend::content_elements">
+                        <resource id="Magento_Cms::media_gallery">
```

Comment:
@Nazar65 let's move the media gallery ACL Resources to the proper namespace: `Magento_MediaGallery::<resource>` and move the `acl.xml` to the `Magento_MediaGallery` module

Reply:
ok